spotinst/ocean_aks_virtual_node_group_auto_scaling/fields_spotinst_ocean_aks_virtual_node_group_auto_scaling.go

```diff
@@ -19,9 +19,8 @@ func Setup(fieldsMap map[commons.FieldName]*commons.GenericField) {
 			Elem: &schema.Resource{
 				Schema: map[string]*schema.Schema{
 					string(Headrooms): {
-						Type:     schema.TypeList,
+						Type:     schema.TypeSet,
```

Comment:
need to make sure it is not a breaking change for users that already have `autoscale_headroom` in their configuration